src/system_window.cpp

```diff
@@ -788,4 +814,82 @@ void SystemWindow::UpdateBrightness()
 	}
 }
 
-} // namespace PanzerChasm
+bool SystemWindow::ScreenShot( const std::filesystem::path& file ) const
+{
+	static uint8_t slot_number = 0;
```

Comment:
It is a bad idea to use de-facto global counter for screenshots names. With such approach older screenshots will be overwritten after game re-start. I suggest to use timestamp-based screenshots naming instead.